test/e2e/rte.go

```diff
@@ -171,3 +193,12 @@ func cmpResourceList(expected, got v1.ResourceList) (string, int, bool) {
 	}
 	return "", 0, true
 }
+
+func execCmd(cmdline []string) ([]byte, error) {
```

Comment:
if we change the test to check the output is a valid version, this can and should be inlined in the calling site.